### PR TITLE
Fixes #714

### LINKED ISSUE #714: document how to deal with OpenShift quotas, especially on OpenShift Online Starter
In our getting started docs, we show how to launch boosters to OpenShift Online, but (AFAIK) we don't tell that OSO has some stringent quotas (especially Starter).

I think the docs should lead users to 1. create a project using Launch (by "project", I mean OpenShift project, or namespace), 2. playing with it, 3. deleting the project before trying to launch another one. That way, one can be sure that nothing is left behind.

We can also list all kinds of resources that Launch creates in the OpenShift project: deployment + pods, service, route, build config + builds, and maybe more depending on the booster (e.g. ConfigMap). These can be deleted manually -- but that's probably less convenient.

CC @quintesse

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

File: docs/topics/proc-getting-started-create-booster.adoc [hunk 1]
```diff
@@ -20,7 +20,11 @@ You can quickly create, build, and deploy a booster to {OpenShiftOnline} using t
 ** Use the _Build and Run Locally_ to create a ZIP download containing your booster. This will *not* create a GitHub repository or deploy your booster to {OpenShiftOnline}.
 . Follow the instructions to create a booster of your choice.
 . Open the `README.adoc` file in your booster's project and follow the instructions for building, deploying, and interacting with your booster.
+. Once you are done interacting with and xref:update-booster[updating] your booster, you should consider link:https://docs.openshift.com/online/dev_guide/projects.html#delete-a-project[deleting your booster project in {OpenShiftOnline}^]. You can redeploy this booster later using the instructions in the xref:oso-create-booster-related-info[booster runtime guide].
++
+You have a link:https://docs.openshift.com/online/dev_guide/compute_resources.html#dev-quotas[quotas^] for your {OpenShiftOnline} account. If you exceed your account quota, you will not be able to launch new boosters using the {launcher} application. The specific for your account will vary depending on your subscription.
```
Comment (Ladicek) [inline docs/topics/proc-getting-started-create-booster.adoc]: "You have a quotas" sounds wrong to me.

Comment (rhoads-zach) [reply docs/topics/proc-getting-started-create-booster.adoc]: nice catch @Ladicek,  fixed in update

**Comment by rhoads-zach** [comment]
@tradej left feedback on your PR

**Comment by tradej** [comment]
@rhoads-zach, since the reviews in #714 have been positive, may I merge?

**Comment by rhoads-zach** [comment]
thanks @tradej @Ladicek @quintesse.

merging.

**Comment by rhoads-zach** [comment]
merged: a2a43a17de6e5cdf88e17194e30389d42309c0c6

**Pull request closed without merging**